### PR TITLE
NMS-13456: Update table formatting in collectors section of docs

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13456

### CODE REVIEW

**Comment by Bonrob2** [comment]
Creating draft PR for content review. I need to determine why this branch can't merge with Foundation 2021.

File: docs/modules/operation/pages/performance-data-collection/collectd/configuration.adoc [hunk 1]
```diff
@@ -1,22 +1,22 @@
 
 [[ga-collectd-configuration]]
-= Configuring _Collectd_ 
+= Configuring Collectd
```
Comment (mmahacek) [inline docs/modules/operation/pages/performance-data-collection/collectd/configuration.adoc:3]:
```suggestion
= Configuring Collectd
```

File: docs/modules/operation/pages/performance-data-collection/collectd/configuration.adoc [hunk 2]
```diff
@@ -1,22 +1,22 @@
```
Comment (mmahacek) [inline docs/modules/operation/pages/performance-data-collection/collectd/configuration.adoc:1]: Combine lines 26 and 27 into one line to match how it shows in the default collectd-configuration.xml

**Review by indigo423 — CHANGES_REQUESTED** [review]
I've build the docs in this branch and wasn't able to find the change. It turned out we have two files:

```
find . -name "collection-packages.adoc"
./modules/operation/pages/performance-data-collection/collection-packages.adoc
./modules/operation/pages/performance-data-collection/collectd/collection-packages.adoc
```

The one you have changed is not included anywhere but I've seen references to it. Seems like we have a bug here:

```
rg "performance-data-collection/collectd/collection-packages.adoc"
modules/operation/pages/meta-data.adoc
172:* <<performance-data-collection/collectd/collection-packages.adoc#ga-collectd-packages-services-meta-data, Performance Management>>

modules/operation/pages/performance-data-collection/collectors/collectors.adoc
8:* collector-specific parameters (used in the <<performance-data-collection/collectd/collection-packages.adoc#ga-collectd-packages,_collectd-configuration.xml_ file>>)

modules/operation/pages/performance-data-collection/collectd/collection-packages.adoc
14:* <<performance-data-collection/collectd/collection-packages.adoc#ga-coll-pack-attribute,collector package attributes>>
15:* <<performance-data-collection/collectd/collection-packages.adoc#ga-collectd-packages-services, collector package services>>
```

**Comment by Bonrob2** [comment]
Thanks for finding that, @indigo423. I will make the required changes.

**Comment by mmahacek** [comment]
Should the SnmpCollector file be moved up a folder?  It's the only collector with its own subfolder.

**Comment by mmahacek** [comment]
> Should the SnmpCollector file be moved up a folder? It's the only collector with its own subfolder.

Okay, I'm seeing some other files that appear to have some odd folder structure. Maybe a review of that once content has been verified.

**Comment by mmahacek** [comment]
Is there a specific sort order to the collectors?  We may want to resort them alphabetically.

**Comment by Bonrob2** [comment]
> Is there a specific sort order to the collectors? We may want to resort them alphabetically.

Alphabetically makes sense, unless there is a reason for them to be in the order they are. Is SNMP used the most?

**Comment by mmahacek** [comment]
> > Is there a specific sort order to the collectors? We may want to resort them alphabetically.
> 
> Alphabetically makes sense, unless there is a reason for them to be in the order they are. Is SNMP used the most?

I don't have any data to back up usage, but I would think SNMP would be up there.

**Comment by Bonrob2** [comment]
@mmahacek and @indigo423 ... includes Mark's updates and resolved conflicts. Ready to merge after review.

**Comment by mmahacek** [comment]
which `collection-packages.adoc` file is the correct one?  The one in the performance-data-collection folder or the one in the collectd folder? I personally would like to see one of them removed in this PR to avoid future confusion.
Also, the following collector docs don't seem to have been touched yet. My initial read over didn't flag any specifics to change, but wanted to check if others had looked at them.
* resource-types.adoc
* property-extenders.adoc
* all the files in the shell subfolder

**Comment by Bonrob2** [comment]
> which `collection-packages.adoc` file is the correct one? The one in the performance-data-collection folder or the one in the collectd folder? I personally would like to see one of them removed in this PR to avoid future confusion.
> Also, the following collector docs don't seem to have been touched yet. My initial read over didn't flag any specifics to change, but wanted to check if others had looked at them.
> 
>     * resource-types.adoc
> 
>     * property-extenders.adoc
> 
>     * all the files in the shell subfolder

Oops. Meant to update that. I didn't review the other collector docs, as they did not have the table issue. I will do that now, and get rid of the duplicate collection-packages.adoc file.

**Comment by Bonrob2** [comment]
> > which `collection-packages.adoc` file is the correct one? The one in the performance-data-collection folder or the one in the collectd folder? I personally would like to see one of them removed in this PR to avoid future confusion.
> > Also, the following collector docs don't seem to have been touched yet. My initial read over didn't flag any specifics to change, but wanted to check if others had looked at them.
> > ```
> > * resource-types.adoc
> > 
> > * property-extenders.adoc
> > 
> > * all the files in the shell subfolder
> > ```
> 
Removed duplicate collection-packages.adoc file. property-extenders and shell files require more work to update. We should capture that work in a different Jira, as there are no tables in those files. Part of a bigger refactoring of the docs.

**Comment by Bonrob2** [comment]
Thanks for the review @mmahacek . Okay to merge?

File: docs/modules/operation/nav.adoc [hunk 3]
```diff
@@ -127,8 +127,9 @@
 ** xref:application-perspective-monitoring/application-perspective-monitoring.adoc[]
 
 * xref:performance-data-collection/introduction.adoc[]
-** xref:performance-data-collection/collection-packages.adoc[]
+** xref:performance-data-collection/collectd/collection-packages.adoc[]
 ** xref:performance-data-collection/collectd/configuration.adoc[]
+** xref:performance-data-collection/resource-types.adoc[]
 ** xref:performance-data-collection/collectors/collectors.adoc[]
 *** xref:performance-data-collection/collectors/snmp/SnmpCollector.adoc[]
```
Comment (mmahacek) [inline docs/modules/operation/nav.adoc:134]: I would suggest moving the SnmpCollector file into the same folder as the rest of the collectors and eliminating the snmp folder.

**Review by mmahacek — APPROVED** [review]
I have one last organization suggestion, but otherwise I think this is good to merge. 👍

File: docs/modules/operation/nav.adoc (hunk 3 above)
Comment (Bonrob2) [reply docs/modules/operation/nav.adoc:134]: You read my mind. I was going to do this, and realized it might break links. But probably better to fix it!

Comment (Bonrob2) [reply docs/modules/operation/nav.adoc]: @mmahacek , updated and alphabetized.

**Pull request merged**